### DISCUSSION TITLE
Is it possible to have a Resharper Rust or something?

### DISCUSSION THREAD

**Discussion by crlf0710** [intellij-rust/intellij-rust#9125] [Q&A] [answered]
I work at a C++ firm, and the firm has bought Visual Studio subscription licenses. We do have some interest in trying out Rust at certain scenerios to see how it goes, but it's hard to make a switch from Visual Studio to CLion at this early point. I'm curious whether it is feasible that Intellij-Rust be integrated as part of a Resharper Rust product or something, that allows people to try it out within existing Visual Studio environments?

**Comment by Undin** [comment, marked as answer]
Hi!
Unfortunately, it's not possible since they use different technologies under the hood (C# and Java/Kotlin) and different API because they target different platforms
